charts/veritable-ui/values.yaml

```
@@ -0,0 +1,643 @@
+  secret: "companysHouseApiKey"
+  existingSecret: ""
+  existingSecretKey: ""
+
+companysHouseApiUrl: "https://api.company-information.service.gov.uk"
+logLevel: "debug"
+
+cookieSessionKeys:
+  enabled: true
+  secret:
+    - "secret"
+  existingSecret: ""
+  existingSecretKey: ""
+
+publicUrl: "http://localhost:3000"
+apiSwaggerBgColor: "#fafafa"
+apiSwaggerTitle: "Veritable"
+apiSwaggerHeading: "Veritable"
+
+idpClientId: "veritable-ui"
+idpPublicURLPrefix: "http://localhost:3080/realms/veritable/protocol/openid-connect"
+idpInternalURLPrefix: "http://localhost:3080/realms/veritable/protocol/openid-connect"
+idpAuthPath: "/auth"
+idpTokenPath: "/token"
+idpJWKSPath: "/certs"
+
+## Digital Catapult veritable-ui image
+## ref: https://hub.docker.com/r/digicatapult/veritable-ui/tags/
+## @param image.registry veritable-ui image registry
+## @param image.repository veritable-ui image repository
+## @param image.tag veritable-ui image tag (immutable tags are recommended)
+## @param image.digest veritable-ui image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag image tag (immutable tags are recommended)
+## @param image.pullPolicy veritable-ui image pull policy
+## @param image.pullSecrets veritable-ui image pull secrets
+## @param image.debug Enable veritable-ui image debug mode
+##
+image:
+  registry: docker.io
+  repository: digicatapult/veritable-ui
+  tag: v0.3.7
+  digest: ""
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+  ## Enable debug mode
+  ##
+  debug: false
+## @param replicaCount Number of veritable-ui replicas to deploy
+##
+replicaCount: 1
+## @param containerPorts.http veritable-ui HTTP container port
+##
+containerPorts:
+  http: 3000
+## Configure extra options for veritable-ui containers' liveness and readiness probes
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+## @param livenessProbe.enabled Enable livenessProbe on veritable-ui containers
+## @param livenessProbe.path Path for to check for livenessProbe
+## @param livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+## @param livenessProbe.periodSeconds Period seconds for livenessProbe
+## @param livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+## @param livenessProbe.failureThreshold Failure threshold for livenessProbe
+## @param livenessProbe.successThreshold Success threshold for livenessProbe
+##
+livenessProbe:
+  enabled: false
+  path: /health
```

```suggestion
livenessProbe:
  enabled: enabled
  path: /
```

For now we will just do a probe off of the root.